packages/embark/src/cmd/cmd_controller.js

```diff
@@ -235,15 +238,19 @@ class EmbarkController {
 
           engine.events.request("webserver:start")
```

Comment:
There are two `engine.events.request("webserver:start")` executions in this method.

Reply:
I don't see the second one, can you point me to it?